ckanext/switzerland/plugin.py

```diff
@@ -140,6 +146,20 @@ class OgdchLanguagePlugin(plugins.SingletonPlugin):
     """
     Handles language dictionaries in data_dict (pkg_dict).
     """
+    plugins.implements(plugins.IConfigurer)
+
+    # IConfigurer
+
+    def update_config(self, config):
+        try:
+            with open(os.path.join(__location__, 'mapping.yaml'),
+                      'r') as format_mapping_file:
+                self.format_mapping = yaml.safe_load(format_mapping_file)
```

Comment:
This might fail with a [YAMLError](https://pyyaml.org/wiki/PyYAMLDocumentation#yamlerror), maybe you can catch both IOError and YAMLError below